spectrum_io/search_result/sage.py

```diff
@@ -33,11 +33,11 @@ def read_result(self, tmt_labeled: str = "") -> pd.DataFrame:
         df.columns = df.columns.str.upper()
         df.columns = df.columns.str.replace(" ", "_")
 
-        df = Sage.update_columns_for_prosit(df, tmt_labeled)
+        df = Sage.update_columns_for_prosit(df, tmt_labeled, custom_mods)
```

Comment:
accordingly here, you have custom static and var mod dictionaries, not just one dictionary.